packages/recommend-core/src/getFrequentlyBoughtTogether.ts

```diff
@@ -31,9 +32,14 @@ export function getFrequentlyBoughtTogether<TObject>({
   return recommendClient
     .getFrequentlyBoughtTogether<TObject>(queries)
     .then((response) =>
-      mapToRecommendations({
+      // Multiple identical recommended `objectID`s can be returned by
+      // the engine, so we need to remove duplicates.
```

Comment:
can that be solved on the API side in the future?

Reply:
cc @jojva for visibility

Reply:
I'm not really aware of this limitation, and knowing the code I'm actually surprised that would be the case.

Reply:
I'll have to dig that a bit

Reply:
I confirm we never return duplicated `objectID`s, you can remove this logic.